### PR TITLE
feat: password reset flow

### PR DESCRIPTION
**Pull request by iuioiua**
Able to take a look, @thorwebdev?

Todo:
- [x] Documentation

Supersedes #105.

### CODE REVIEW

**Comment by thorwebdev** [comment]
The code looks good to me, the only thing is, when I test it on my project, I don't receive the email, and I currently don't know why. Is it working for you?

**Comment by iuioiua** [comment]
Yeah, it worked the last time I tried. Let me try again.

**Comment by iuioiua** [comment]
I'll go ahead and merge this PR as it's working in the preview deployment. We can explore the testing of the password reset flow in #157.

image
